### ISSUE TITLE
Game content not loading anything

### ISSUE THREAD

**Issue by GForce134** [fgl27/SmartTwitchTV#208]
Game content not loading any streams

Any game you select only gives you a blank page

The page should give streams related to the game.
 
 **Device:**
 Chromecast 4k with google tv

**Comment by fgl27** [comment]
what game and what language have you selected?

as I can't reproduce it, it must be a specific game or language.

**Comment by fgl27** [comment]
and you are not even see the banner? this

![image](https://github.com/fgl27/SmartTwitchTV/assets/13755892/0464569a-53d4-42f7-af9f-4cfcbfde213e)

**Comment by GForce134** [comment]
Yes I can see this banner but when I check the same game content through the official twitch app there are always multiple streams. Even for streamers I'm subscribed to, it says they are streaming for example game content iRacing but when I go to the iRacing section they don't show up there.


iRacing is the game I usually ecounter this with and I use default english language.


Thank you for your amazing work!

**Comment by fgl27** [comment]
iRacing in the English language there are only 13 streamers right now, there is a chance that when you look in that language there were none, very unpopular game

![image](https://github.com/fgl27/SmartTwitchTV/assets/13755892/22e927cc-9e53-4ac6-bca8-e864760e80ff)

The game section begins only live streams for that game and language, you can use language all to see more

this is the language all, the official app doesn't filter by language so that is how you have to search

![image](https://github.com/fgl27/SmartTwitchTV/assets/13755892/ba874af2-e62f-4acf-9d58-0cf7828e0e81)

**Comment by Hardcor3UK** [comment]
Suddenly getting this/similar issue last few days, game list itself loads fine, but clicking on a game get stuck with inifinite loading.

Viewing the web app and inspecting the network traffic seems like OAuth failure on the API.

Seems like API problem specific to game listings when not logged in, activating an account in the app fixes the issue for me.

**Comment by fgl27** [comment]
> Suddenly getting this/similar issue last few days, game list itself loads fine, but clicking on a game get stuck with inifinite loading.
> 
> Viewing the web app and inspecting the network traffic seems like OAuth failure on the API.
> 
> Seems like API problem specific to game listings when not logged in, activating an account in the app fixes the issue for me.

you are not seeing this?

![image](https://github.com/user-attachments/assets/672d6d54-b365-4a39-9f61-0c56f18aa11c)

**Comment by Hardcor3UK** [comment]
no i'm seeing the loading circle constantly and it never finishes

![image](https://github.com/user-attachments/assets/dfc89c71-f2c7-4c67-b33a-62464459dd4e)

**Comment by fgl27** [comment]
What is that device, computer?

**Comment by fgl27** [comment]
OK I got it

all OK now

![image](https://github.com/user-attachments/assets/0e03daa2-5bdf-4620-b4a1-782a59e89d62)
